recipes/argparse/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
   "2.1":
     url: "https://github.com/p-ranav/argparse/archive/v2.1.tar.gz"
-    sha256: "0a82f464b568b8ee6650fc837f371eb9c81417e2ef9fb3b51f65ad50fa3b8662"
+    sha256: "44f40a5b9908e722a419973f676bfad4f34d904bf23f89351c477988bc5c4044"
```

Comment:
The former tag was restored. This change is not needed anymore

https://github.com/p-ranav/argparse/issues/126#issuecomment-919211321

Reply:
does argparse author need to trigger something so new 2.2 is uploaded to conan-center?

Reply:
Anyone can add new versions via pull requests. It does not have to be the project maintainers themselves

Reply:
got it. Let me give it a try. I need to get an approval from my company, Qualcomm before I go ahead and submit. But maybe someone will beat me to it.

Reply:
what is this sha256? where do I take it from?

Reply:
OK. I think this is sha256sum result. got it.